Tests/ArcGISToolkitTests/ARTests.swift

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if !os(visionOS)
+#if !os(visionOS) || !targetEnvironment(macCatalyst)
 import ArcGIS
 import SwiftUI
 import XCTest
 @testable import ArcGISToolkit
 
+@available(macCatalyst, unavailable)
```

Comment:
This is no longer needed
```suggestion
```

Reply:
Even with `#if !os(visionOS) || !targetEnvironment(macCatalyst)`, without this in place the tests still show from the Test Navigator in Xcode with Catalyst as the destination and causes errors when you try to run them:

<p align="center">
  <img width="50%" src="https://github.com/user-attachments/assets/47b7fc20-c3f7-4781-9501-4f8d1f3a130a">
</p>

With `@available(macCatalyst, unavailable)` in place, they still show in Test Navigator but don't run.